apps/website/src/examples/Checkbox.indeterminate.tsx

```diff
@@ -0,0 +1,58 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { Checkbox, Flex } from '@itwin/itwinui-react';
+
+export default () => {
+  const [option1, setOption1] = React.useState(false);
+  const [option2, setOption2] = React.useState(true);
+  const [allOptions, setAllOptions] = React.useState(false);
```

Comment:
I've just realized that allOptions also doesn't need to be a state.

You can derive it from option1 and option2:
```tsx
const allOptions = option1 && option2;
```

Doing this will also allow you remove the whole useEffect from lines 14 to 24.

Same comment in Checkbox.visibility

Reply:
Change is made in both indeterminate & visibility demos.